test/e2e/knative_serving_test.go

```diff
@@ -73,6 +73,25 @@ func TestKnativeServing(t *testing.T) {
 		}
 	})
 
+	t.Run("verify kn ConcoleCLIDownload CO and deployment resources", func(t *testing.T) {
+		// Check the status of Deployment for kn ConsoleCLIDownload
+		if err := test.CheckDeploymentScale(caCtx, knativeServing, "kn-cli-downloads", "1"); err != nil {
+			t.Fatalf("failed to verify kn ConcoleCLIDownload deployment: %v", err)
+		}
+		// Verify that Route for kn ConsoleCLIDownload is ready and has a host
+		host, err := test.RouteHost(caCtx, knativeServing, "kn-cli-downloads")
+		if err != nil {
+			t.Fatalf("failed to verify kn ConcoleCLIDownload route: %v", err)
+		}
+		if host == "" {
```

Comment:
This can never happen, I believe?

Reply:
Yup, if Route is ready its Ingress got to have a `.Host`.